subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/logging/GroupedNodeFixture.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.logging;
+
+import org.gradle.integtests.fixtures.logging.comparison.ExhaustiveLinesSearcher;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.gradle.util.internal.CollectionUtils.filter;
+import static org.gradle.util.internal.CollectionUtils.join;
+
+public abstract class GroupedNodeFixture {
```

Comment:
💭 Maybe just `AbstractGroupedFixture`?  I think Node is misleading.  And javadoc.

Reply:
Changes were merged in a separate PR: https://github.com/gradle/gradle/pull/23931